lametro/people.py

```diff
@@ -83,7 +83,8 @@ def scrape(self):
 
         for body in self.bodies():
             if body['BodyTypeId'] == body_types['Committee']:
-                o = Organization(body['BodyName'],
+                org_name = body['BodyName'].strip()
```

Comment:
`strip()` will nab the whitespace from either side of the string. If leading whitespace isn't meaningful, this is fine. If you just want the trailing whitespace, use [`rstrip()`](https://docs.python.org/3.5/library/stdtypes.html?highlight=strip#str.rstrip).